.travis.yml

```diff
@@ -14,3 +14,9 @@ before_script:
 script:
   - mvn verify
   - sudo cat /var/log/tarantool/jdk-testing.log
+
+after_success:
+  - |
+    if [ "${TRAVIS_JDK_VERSION}" = openjdk11 ]; then
+        mvn clean verify jacoco:report coveralls:report -DrepoToken=${COVERALLS_TOKEN}
+    fi
```

Comment:
I maybe missing something, but I guess one can save time and collect coverage during the "script" phase and only upload results to coveralls.io on "after_success" step. Something along these lines: 
```
script:
   - mvn verify jacoco:report

after_success:
   - mvn coveralls:report -DrepoToken=${COVERALLS_TOKEN}
```

Reply:
Thanks. Fixed.